src/bind.js

```diff
@@ -18,8 +18,9 @@
  *
  * @return {Function}
  */
-function bind(callback) {
-  // write code here
-}
+
+const bind = (callback, ...firstArg) =>
+  (...args) =>
+    callback(...firstArg, ...args);
```

Comment:
for better readability this version is better
```
const bind = (callback, ...firstArg) => {
  return (...args) => callback(...firstArg, ...args)
}
```

Reply:
if you want to place arrow function body on the next line, wrap it with parentheses